openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/util/ORSParameters.java

```diff
@@ -25,6 +25,14 @@ private ORSParameters() {}
     /* Parameters with an 'INIT' prefix are used as defaults and/or are configured at start.*/
     static final String ROUTING_INIT_PREFIX = "routing.";
 
+    /**
+     * Parameters that can be passed as hints and influence routing per request.
+     */
+    public static final class Routing {
```

Comment:
I find `Routing` as class name a bit confusing here. This should be usable for matrix, isochrones too, correct? Maybe something like `Weighting`? Or `Calculation`? Idk

Reply:
Thanks, done!